clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2579,6 +2591,39 @@ def CIR_FuncOp : CIR_Op<"func", [
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// LLVMIntrinsicCallOp
+//===----------------------------------------------------------------------===//
+
+def CIR_LLVMIntrinsicCallOp : CIR_Op<"llvm.intrinsic"> {
```

Comment:
```suggestion
def CIR_LLVMIntrinsicCallOp : CIR_Op<"call_llvm_intrinsic"> {
```
The name used in the incubator is quite confusing. Generally, in MLIR operations, I expect `<dialect>.<op>` to represent a dialect qualified operation, so something like `cir.llvm.intrinsic` doesn't parse right in my mind since `cir` and `llvm` both look like dialect qualifiers.